src/librustc_mir/diagnostics.rs

```diff
@@ -1293,26 +1296,27 @@ fn outside_closure(x: &mut i32) {
 }
 
 fn foo(a: &mut i32) {
-    let bar = || {
+    let mut bar = || {
         inside_closure(a)
     };
     outside_closure(a); // error: cannot borrow `*a` as mutable because previous
                         //        closure requires unique access.
+    bar();
 }
 ```
 
-To fix this error, you can place the closure in its own scope:
+To fix this error, you can place finish using the closure in its own scope:
```

Comment:
are you just trying to show off NLL here? The original suggestion (of putting the closure "in its own scope") would still *work*, right?

Reply:
Well, there's now a usage of the closure. Adding a scope would either be extraneous, or result in a name resolution error depending on how one interprets it.